sources/packages/web/src/components/students/modals/ConfirmChangeApplication.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <modal-dialog-base title="Edit application" :showDialog="showDialog">
+    <template #content>
+      <v-checkbox
+        v-model="conditionsAccepted"
+        label="I acknowledge that by editing my application, I may be required to
```

Comment:
Please consider some standard spacing between the Acknowledgement and Banner. 
![image](https://github.com/user-attachments/assets/6fe40ef9-9b00-41b2-b810-c5d14c65d4d3)